pyproject.toml

```diff
@@ -11,9 +11,9 @@ license = "Apache 2.0"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "<3.11,>=3.7.1"
+python = "<3.12,>=3.8"
```

Comment:
breaking change

Reply:
I wouldn't hard bump the version number for this but 🤷

Reply:
Newer versions of the SDK no longer support 3.7.